mypy.ini

```diff
@@ -0,0 +1,3 @@
+[mypy]
+ignore_missing_imports = True
+warn_unused_ignores = True
```

Comment:
Having a new line at the end of file is always good :)

Reply:
valid